packages/shared/locales/en.json

```
@@ -2405,6 +2406,13 @@
                 "Conflicting": "Failed to create account"
             }
         },
+        "delegation": {
+            "creation": {
+                "Pending": "creating delegation",
+                "Confirmed": "delegation created",
```

Feel free to update `account.creation` as well if you wish to
```suggestion
                "Pending": "Creating delegation",
                "Confirmed": "Delegation created",
```